contents/docs/cdp/airbyte-export.md

```diff
@@ -10,7 +10,7 @@ This Airbyte Export app sends data from PostHog, to Airbyte. It supports both Fu
 
 ## What are the requirements for this app?
 
-Using the Airbyte Export app requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.
+Using the Airbyte Export app requires either PostHog Cloud with the data pipeline add-on, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.
```

Comment:
nit: might be good to link to billing page for these ones too - maybe it can be an imported snippet for all of them